pkg/app/couchbase.go

```diff
@@ -129,7 +132,7 @@ func (cb *CouchbaseDB) Object() crv1alpha1.ObjectReference {
 	return crv1alpha1.ObjectReference{
 		APIVersion: "v2",
 		Group:      "couchbase.com",
-		Name:       fmt.Sprintf("%s-couchbase-cluster", cb.chart.Release),
+		Name:       fmt.Sprintf("%s", cb.chart.Release),
```

Comment:
```suggestion
		Name:       cb.chart.Release,
```
This is already a string - will fix the lint failure

Reply:
Done @pavannd1